wordpress-mobile/WordPress-Android
# Added a focused state to cover Header and List toolbar buttons.

Fixes #11701

Adds a missing state to Aztec selector that covers buttons that are focused instead of being selected. 

To test:
- Navigate to Aztec editor, input any Heading and confirms that the corresponding button in the toolbar is selected when the cursor is in the heading.
- Repeat this for ordered or unordered list.
- Confirm that button is correctly highlighted in both light and dark modes.

PR submission checklist:
